fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
I left a comment in that gist but I guess it doesn't announce on those:

    should unique names be reported instead? May be with counts

    side note: we should really come up with Json output some time